doc/source/merging.rst

```diff
@@ -353,7 +353,40 @@ Passing ``ignore_index=True`` will drop all name references.
 More concatenating with group keys
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Let's consider a variation on the first example presented:
+A fairly common use of the ``keys`` argument is to override the column names when creating a new DataFrame based on existing Series.
+Notice how the default behaviour consists on letting the resulting DataFrame inherits the parent Series' name, when these existed.
+
+.. ipython:: python
+
+   s3 = Series(['_0', '_1', '_2', '_3'], name='foo')
+   s4 = Series(['_0', '_1', '_2', '_3'])
+   s5 = Series(['_0', '_1', '_4', '_5'])
+
+   result = concat([s3, s4, s5], axis=1)
+
+.. ipython:: python
+   :suppress:
```

Comment:
figures are not needed for this section. Its a discussion of naming via ``keys``. pls remove.